chain/consensus/filcns/upgrades.go

```diff
@@ -302,6 +303,17 @@ func DefaultUpgradeSchedule() stmgr.UpgradeSchedule {
 		Height:    build.UpgradeCalibrationDragonFixHeight,
 		Network:   network.Version22,
 		Migration: upgradeActorsV13VerifregFix(calibnetv13BuggyVerifregCID1, calibnetv13CorrectManifestCID1),
+	}, {
+		Height:    build.UpgradeAussieHeight,
+		Network:   network.Version23,
+		Migration: UpgradeActorsV14,
+		PreMigrations: []stmgr.PreMigration{{
+			PreMigration:    PreUpgradeActorsV14,
+			StartWithin:     120,
+			DontStartWithin: 15,
+			StopWithin:      10,
+		}},
+		Expensive: true,
```

Comment:
I don't know exactly what this flag is for but I suspect it would be better being set to `false` since we're only planning O(1) migration work on state this migration.

Reply:
Its not entirely clear to me what this flag does either. But I can see that all other migrations have had set this flag to expensive.

I will be asking around, to see if I can find some answers to this, and then retroactively set it to `false`.